packages/tasks/src/tasks/text-generation/data.ts

```diff
@@ -58,8 +62,8 @@ const taskData: TaskDataCustom = {
 	],
 	models: [
 		{
-			description: "A text-generation model trained to follow instructions.",
-			id: "google/gemma-2-2b-it",
+			description: "Smaller variant of one of the most powerful models.",
+			id: "deepseek-ai/DeepSeek-R1-Distill-Qwen-1.5B",
```

Comment:
to avoid repetition of the orgs, maybe we can keep it as-is w/ Gemma or Llama 3.2 1B

probably Gemma since it's still used quite a bit.